client/src/components/Header/Header.scss

```diff
@@ -1,37 +1,86 @@
+@import "../../../node_modules/bootstrap/scss/functions";
+@import "../../../node_modules/bootstrap/scss/variables";
+@import "../../../node_modules/bootstrap/scss/mixins";
+
 .header {
     display: flex;
-    padding: 2rem;
     font-size: 0.875rem;
     position: relative;
+    padding: 1rem;
+    @include media-breakpoint-up(lg) {
```

Comment:
@bxsmith007 this is how you control responsiveness from SCSS using Bootstrap. The other way is the Bootstrap classes, such as `d-none d-lg-block` which would make something `display: none` by default and `display: block` on large or greater screen sizes https://getbootstrap.com/docs/4.0/utilities/display/